src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -62,6 +63,34 @@ class _User:
     """The bidirectional channel to communicate with the user."""
 
 
+@dataclass
+class _CacheEntry:
+    """Represents an entry in the cache with expiry time."""
+
+    inv_bat_pair: InvBatPair
+    """The inverter and adjacent battery data pair."""
+
```

Comment:
I confirm the gap discharge is set to 2 hours in peak-shaving actor, so the default duration is set to 2.5 hours as it was initially suggested